runconfig/opts/parse_test.go

```diff
@@ -186,7 +186,7 @@ func TestDecodeContainerConfigVolumes(t *testing.T) {
 		t.Fatalf("binds %v should have failed", bindsOrVols)
 	}
 	if _, _, err := callDecodeContainerConfig(bindsOrVols, nil); err == nil {
-		t.Fatalf("binds %v should have failed", bindsOrVols)
+		t.Fatalf("volume %v should have failed", bindsOrVols)
```

Comment:
The `bindsOrVols` variable in this case holds a path on the host (`/tmp` or `C:\temp\`), so this is testing a *bind*, not a *volume*